### PR TITLE
Improve Ruby SDK's example code for before_send callback

### PR DESCRIPTION
**Pull request by st0012**

### CODE REVIEW

File: src/includes/configuration/before-send/ruby.mdx [hunk 1]
```diff
@@ -1,7 +1,18 @@
 ```ruby
 Sentry.init do |config|
-  config.before_send = lambda do |event, hint|
-    event[:key] = value
+  config.before_send = lambda do |event, _hint|
+    # note1: if you have config.async configured, the event here will be a Hash instead of an Event object
+    # note2: the code below is just an example, you should adjust the logic base on your needs
```
Comment (PeloWriter) [inline src/includes/configuration/before-send/ruby.mdx:5]:
```suggestion
    # note2: the code below is just an example, you should adjust the logic based on your needs
```

**Review by PeloWriter — APPROVED** [review]
looks good. Thanks.

**Comment by st0012** [comment]
thanks for the review, have updated 🙂

**Comment by PeloWriter** [comment]
Thanks again. Are we waiting on further approval? If not, do you want me to merge, or will you?

**Comment by st0012** [comment]
@PeloWriter I don't have the permission to merge PRs in this repo 🙂

**Comment by PeloWriter** [comment]
Done :) Thanks.

**Pull request merged**